### PR TITLE
Campsy JIRA integration

### PR DESCRIPTION
**Pull request by artemstepanenko**

### CODE REVIEW

File: server/src/routes/jiraIssues.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  username: config.jira.username,
+  password: config.jira.password,
+  apiVersion: '2',
+  strictSSL: true,
+});
+
+const router = express.Router();
+
+router.get('/', (req, res, next) => {
+  jira.searchJira("project = CC AND status in (Open, 'In Progress', Reopened)")
```
Comment (milenvoutchev) [inline server/src/routes/jiraIssues.js]: As the route is something between Controller and View, please move as much as possible data-fetching and business logic to a separate "Service". 
Keep only the response building here.

File: server/src/routes/jiraIssues.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        if (priority === 'Blocker') {
+          blockers += 1;
+        } else if (priority === 'Critical') {
+          criticals += 1;
+        } else {
+          others += 1;
+        }
+      });
+
+      res.json({
```
Comment (milenvoutchev) [inline server/src/routes/jiraIssues.js]: For the response, use `getResponseSuccess` (newly added :)), e.g.:
```
import { getResponseSuccess } from '../helper/responseHelper';
[...]
res.json(getResponseSuccess(payload));
```

Comment (milenvoutchev) [inline server/src/routes/jiraIssues.js]: Also, please add caching layer, e.g. https://github.com/rocket-internet-berlin/RocketDashboard/blob/master/server/src/routes/newRelicErrors.js#L32

Comment (artemstepanenko) [inline server/src/routes/jiraIssues.js]: Fixed.

File: server/src/service/googleService.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+  constructor(serviceAccountEmail, serviceAccountPrivateKey) {
+    this.jwtClient = new google.auth.JWT(
+      serviceAccountEmail,
+      null,
+      serviceAccountPrivateKey,
+      ['https://www.googleapis.com/auth/spreadsheets.readonly'],
+      null,
+    );
+    this.firstRow = 3;  // FIXME: magic number
+    this.maxEntriesAmount = 50;  // FIXME: magic number
```
Comment (milenvoutchev) [inline server/src/service/googleService.js]: Both magic numbers can now go to `config.bugsHistory`

File: server/src/service/googleService.js [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+        const columns = data.map((column) => column.rowData.map(cellData => _get(cellData, 'values[0].userEnteredValue')));
+        const history = columns[0].map((timestamp, i) => {
+          const googleTimestamp = timestamp.numberValue;
+          const date = moment(new Date(1899, 12, 30)).add(googleTimestamp, 'd').format('MMM, D'); // conversion from a Google's timestamp format
+          const openBugs = columns[1][i] === undefined ? 0 : columns[1][i].numberValue;
+          const solvedBugs = columns[2][i] === undefined ? 0 : columns[2][i].numberValue;
+          const newBugs = columns[3][i] === undefined ? 0 : columns[3][i].numberValue;
+          return { date, openBugs, solvedBugs, newBugs };
+        });
+        // TODO: Move building of API response to the "route"
```
Comment (milenvoutchev) [inline server/src/service/googleService.js]: Remove TODO comment - not relevant anymore

File: server/src/routes/jiraIssues.js (hunk 2 above)
Comment (artemstepanenko) [inline server/src/routes/jiraIssues.js]: Done.

**Pull request merged**